src/ChipInput.js

```diff
@@ -304,6 +304,7 @@ class ChipInput extends React.Component {
   }
 
   handleKeyUp = (event) => {
+    if (this.state.preventChipCreation) return
```

Comment:
Why is this needed?

Reply:
this is needed to avoid input to be cleared, i tried removing it but this is the result:
![bug-clear-input](https://user-images.githubusercontent.com/4092865/30955942-bb8f8c92-a435-11e7-8bd9-0683ba949ed2.gif)